src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -127,6 +127,7 @@ public static class Features
             public const string TypeInferenceService = "TypeInferenceService";
             public const string Venus = "Venus";
             public const string VsLanguageBlock = "VsLanguageBlock";
+            public const string VsNavInfo = "VsNavInfo";
```

Comment:
Pfftt. What value does that add? :smile: I would want it to actually stay in sync with my feature.